mqtt_client/include/mqtt_client/MqttClient.ros2.hpp

```diff
@@ -242,6 +278,15 @@ class MqttClient : public rclcpp::Node,
    */
   void mqtt2primitive(mqtt::const_message_ptr mqtt_msg);
 
+  /**
+   * @brief Publishes a primitive message received via MQTT to ROS.
+   *
+   * Currently not implemented.
+   *
+   * @param   mqtt_msg     MQTT message
+   */
+  void mqtt2fixed(mqtt::const_message_ptr mqtt_msg);
+
```

Comment:
Guess this docstring was copy-pasted. Also, I believe the `Currently not implemented` is not correct for `void mqtt2primitive` as well.